ports/dcmtk/portfile.cmake

```
@@ -35,6 +55,85 @@ vcpkg_cmake_configure(
 vcpkg_cmake_install()
 vcpkg_copy_pdbs()
 
+if ("tools" IN_LIST FEATURES)
+    set(_tools
+        cda2dcm
+        dcm2json
+        dcm2pdf
+        dcm2pnm
+        dcm2xml
+        dcmcjpeg
+        dcmcjpls
+        dcmconv
+        dcmcrle
+        dcmdata_tests
+        dcmdjpeg
+        dcmdjpls
+        dcmdrle
+        dcmdspfn
+        dcmdump
+        dcmect_tests
+        dcmfg_tests
+        dcmftest
+        dcmgpdir
+        dcmicmp
+        dcmiod_tests
+        dcmj2pnm
+        dcml2pnm
+        dcmmkcrv
+        dcmmkdir
+        dcmmklut
+        dcmnet_tests
+        dcmodify
+        dcmp2pgm
+        dcmprscp
+        dcmprscu
+        dcmpschk
+        dcmpsmk
+        dcmpsprt
+        dcmpsrcv
+        dcmpssnd
+        dcmqridx
+        dcmqrscp
+        dcmqrti
+        dcmquant
+        dcmrecv
+        dcmrt_tests
+        dcmscale
+        dcmseg_tests
+        dcmsend
+        dcmsign
+        dcmsr_tests
+        dcmtls_tests
+        dcod2lum
+        dconvlum
+        drtdump
+        drttest
+        dsr2html
+        dsr2xml
+        dsrdump
+        dump2dcm
+        echoscu
+        findscu
+        getscu
+        img2dcm
+        mkreport
+        movescu
+        msgserv
+        ofstd_tests
+        pdf2dcm
+        stl2dcm
+        storescp
+        storescu
+        termscu
+        wlmscpfs
+        wltest
+        xml2dcm
+        xml2dsr
+    )
+    vcpkg_copy_tools(TOOL_NAMES ${_tools} AUTO_CLEAN)
```

```suggestion
    vcpkg_copy_tools(TOOL_NAMES ${_tools})
```

When you enable tools only for release, you can get rid of `AUTO_CLEAN` here.

---

If I remove AUTO_CLEAN, a warning was given during the build

`The following EXEs were found in /bin or /debug/bin. EXEs are not valid distribution targets.

    D:/vcpkg/packages/dcmtk_x64-windows/bin/cda2dcm.exe
    D:/vcpkg/packages/dcmtk_x64-windows/bin/dcm2json.exe
    ...

Found 1 post-build check problem(s). To submit these ports to curated catalogs, please first correct the portfile:
    D:\vcpkg\ports\dcmtk\portfile.cmake
-- Performing post-build validation done`

After add AUTO_CLEAN build is OK
`-- Fixing pkgconfig file: D:/vcpkg/packages/dcmtk_x64-windows/debug/lib/pkgconfig/dcmtk.pc
-- Installing: D:/vcpkg/packages/dcmtk_x64-windows/share/dcmtk/copyright
-- Performing post-build validation
-- Performing post-build validation done`


Enable BUILD_APPS just for release

---

Looks like additional executables are installed that aren't controlled by `BUILD_APPS`:

```
D:/vcpkg/packages/dcmtk_x64-windows/bin/cda2dcm.exe
D:/vcpkg/packages/dcmtk_x64-windows/bin/dcm2json.exe
```

In this case, you can't remove `AUTO_CLEAN` here.